manifests/type/capactio/capact/validation/storage.yaml

```diff
@@ -34,24 +34,7 @@ spec:
           },
           "contextSchema": {
             "$id": "#/properties/contextSchema",
-            "const": {
-                "$schema": "http://json-schema.org/draft-07/schema",
-                "type": "object",
-                "required": [
-                  "provider"
-                ],
-                "properties": {
-                  "provider": {
-                    "$id": "#/properties/context/properties/provider",
-                    "type": "string",
-                    "enum": [
-                      "aws_secretsmanager",
-                      "dotenv"
-                    ]
-                  }
-                },
-                "additionalProperties": false
-              }
+            "const": "\n\t{\n      \"$schema\": \"http://json-schema.org/draft-07/schema\",\n      \"type\": \"object\",\n      \"required\": [\n        \"provider\"\n      ],\n      \"properties\": {\n        \"provider\": {\n          \"$id\": \"#/properties/context/properties/provider\",\n          \"type\": \"string\",\n          \"enum\": [\n            \"aws_secretsmanager\",\n            \"dotenv\"\n          ]\n        }\n      },\n      \"additionalProperties\": false\n    }\n"
```

Comment:
@mkuziemko Did you test the AWS Secrets Manager storage installation? I suspect that you did not, cause it won't work - you also need to modify the secrets-manager/storage/install.yaml to make sure it uploads proper shape of the TypeInstance `contextSchema. You can base on what I did here: https://github.com/capactio/hub-manifests/pull/63, but please test it before putting it again to review.

Reply:
I'm sorry @mkuziemko, this comment is invalid as the storage TI is properly created on Capact side. Sorry for making confusion
![tenor-22273720](https://user-images.githubusercontent.com/7155799/159505602-c2a6bb75-0d8f-44b4-98c1-e33e64100f2e.gif)